icu4c/source/common/unicode/platform.h

```diff
@@ -821,8 +839,9 @@ namespace std {
 
 #ifdef U_IMPORT
     /* Use the predefined value. */
-#elif defined(_MSC_VER) || (__has_declspec_attribute(dllexport) && __has_declspec_attribute(dllimport))
-    /* Windows needs to export/import data. */
+#elif defined(_MSC_VER) || (UPRV_HAS_DECLSPEC_ATTRIBUTE(dllexport) && \
+                            UPRV_HAS_DECLSPEC_ATTRIBUTE(dllimport))
+/* Windows needs to export/import data. */
```

Comment:
Please restore the indentation of this line.

Reply:
Done